core/vm/gas_table.go

```diff
@@ -95,9 +95,12 @@ var (
 
 func gasSStore(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize uint64) (uint64, error) {
 	var (
-		y, x    = stack.Back(1), stack.Back(0)
-		current = evm.StateDB.GetState(contract.Address(), x.Bytes32())
+		y, x         = stack.Back(1), stack.Back(0)
+		current, err = evm.StateDB.GetState(contract.Address(), x.Bytes32())
 	)
+	if err != nil {
+		return 0, err
```

Comment:
This function tells the VM how much gas to use for opSstore. In the case of an error, zero gas should be used, so 0 is returned.

Reply:
We could also consider how much gas should be used in the error case to prevent a DDoS attack, but it should be minimal (similarly, in opCall, zero gas is used)

Reply:
Ahhh, makes sense. I had just been looking at opCodes consuming gas, and got my wires crossed. Checks out ✅